openedx/adg/lms/course_meta/apps.py

```diff
@@ -0,0 +1,8 @@
+"""
+Configurations for course_meta app
+"""
+from django.apps import AppConfig
+
+
+class CourseMetaConfig(AppConfig):
```

Comment:
Config is defined but is not being used anywhere. To override default config we need to add this line in __init__ of this apps.
`default_app_config = 'path'`